package-examples/tenant/.krmignore

```diff
@@ -0,0 +1 @@
+.snippets
```

Comment:
I think the snippets directory is a little bit controversial for me.  If at all it's possible to do an edit (rm for instance)( or use an imperative function flow, then we don't have to introduce another concept in this workflow until we fully hash it out.

Reply:
Thanks for the feedback.  I was trying to address `optional` resources and exploring a way for platform teams to convey canned configurations in the pkg. I agree that the pattern is not tested and I was hoping to get feedback with this guide :) Do you know adding a note in the guide asking for feedback will help ?

`rm` based workflow which is analogous to `opt-out` which may be the right thing to do in some use-cases.

Reply:
Thought some more about it and I agree that we should probably test the `snippets` idea separately. So removed `snippets` concept completely from this guide and now the workflow is very simple.